domain/learningandworkprogress/src/testFixtures/kotlin/uk/gov/justice/digital/hmpps/domain/CommonBuilders.kt

```diff
@@ -7,10 +7,6 @@ import java.util.UUID
  * prison numbers, times and dates etc.
  */
 
-fun aValidPrisonNumber() = "A1234BC"
-
-fun anotherValidPrisonNumber() = "B5678CD"
```

Comment:
One for another day perhaps, but I was wondering recently about the randomValidPrisonNumber function, and specifically line 22:
```
val numberPart = numbers.random().toString().padStart(4, '0')
```
As I read that (and I could be wrong), I think that generates a 4 digit string, where the first 3 characters are `0`, and only the 4th is a random digit? IE. Its creating a random string number of between `0000` and `0009`
(I think thats what it does anyway 🤷‍♂️ )

I wonder if it should be something as pedestrian as: 
```
val numberPart = "${numbers.random()}${numbers.random()}${numbers.random()}${numbers.random()}"
```
(though theres probably something cleverer than that 😁 )